src/batch-client/index.ts

```diff
@@ -509,6 +536,13 @@ export class ZimbraBatchClient {
 			}
 		});
 
+	public resetPassword = (body: ResetPasswordOptions) =>
```

Comment:
better to destructure body and get password and use it below, it makes code more cleaner and we know how many params are getting passed to api